src/jsonrpc/miner_jsonrpc_dkg.erl

```diff
@@ -62,14 +62,20 @@ handle_rpc(<<"dkg_running">>, []) ->
     Delay = max(0, (Diff div RestartInterval) * RestartInterval),
     case Curr >= Height of
         false ->
-            #{ error => <<"not running">> };
+            #{ running => false };
         _ ->
             {ok, Block} = blockchain:get_block(Height+Delay, Chain),
             Hash = blockchain_block:hash_block(Block),
             ConsensusAddrs = blockchain_election:new_group(Ledger, Hash, N, Delay),
-            [ #{ name => ?BIN_TO_ANIMAL(A),
-                 address => ?BIN_TO_B58(A) }
-              || A <- ConsensusAddrs ]
+            #{
+                running => true,
+                consensus_members => [
+                    #{
+                        name => ?BIN_TO_ANIMAL(A),
+                        address => ?BIN_TO_B58(A)
+                   } || A <- ConsensusAddrs
+                ]
+            }
```

Comment:
Another return-type normalization. `running` will always exist. `consensus_members` will show up if `running` is true.